starlark/eval.go

```diff
@@ -1225,8 +1225,22 @@ func Call(thread *Thread, fn Value, args Tuple, kwargs []Tuple) (Value, error) {
 	fr.callable = c
 
 	thread.beginProfSpan()
+
+	// Use defer to ensure that panics from built-ins
```

Comment:
Should we document that the interpreter is still usable after an application-defined builtin panic()s? Or would this be conventionally assumed in Go?

Reply:
I'd rather not make any promises about what the interpreter does when the client code is buggy. But it's easy enough for us to actually implement this property for now, and both the code here and the test in this CL record the intent for the maintainers.